### ISSUE TITLE
Plasma2D interpolation failing

### ISSUE THREAD

**Issue by Didou09** [ToFuProject/tofu#209]
Affect tofu 1.4.1:

Minimum working example:

```
multi = tf.imas2tofu(shot=54719, ids=['wall', 'pulse_schedule', 'soft_x_rays', 'equilibrium'])
sxr = multi.to_Data('sxr')
cam = sxr.lCam[0]
cam.set_dsino([2.4, 0.])
pts = cam.dsino['pts']
plasma = multi.to_Plasma2D()

plasma.interp_pts2profile(pts=pts, quant='equilibrium.rhotn')
```

**Comment by Didou09** [comment]
After Jorge's investigations it seems the error may come from:
* Wrong assert (not...)
* tall not defined
* wrong default value (None instead of 'nearest')

To be verified

**Comment by Didou09** [comment]
Fixed, in devel, by PR #210
